nemo/collections/asr/data/audio_to_audio.py

```diff
@@ -1094,7 +1131,12 @@ def __init__(
             max_number=max_utts,
         )
 
-        audio_processor = ASRAudioProcessor(sample_rate=sample_rate, random_offset=random_offset,)
+        audio_processor = ASRAudioProcessor(
+            sample_rate=sample_rate,
+            random_offset=random_offset,
+            normalize=normalize_input,
+            normalization_signal='input_signal',
```

Comment:
By default, `normalize_input=False`, but we still always pass a string argument for `normalization_signal`.

Would it make sense to condense both into a single `Optional[str]` argument? If it's `None`, then do nothing, if it's something then normalize as specified.